src/Razor/src/Microsoft.VisualStudio.Mac.RazorAddin/mpack/addin.info

```diff
@@ -43,5 +43,6 @@
   <Dependencies>
     <Addin id="::MonoDevelop.Core" version="17.3" />
     <Addin id="::MonoDevelop.Ide" version="17.3" />
+    <Addin id="TextEditor" version="17.3" />
```

Comment:
This might need to be `::MonoDevelop.TextEditor`. I'll double-check.

Reply:
Yeah, it does, because TextEditor means `Microsoft.VisualStudio.Mac.RazorAddin.TextEditor` I think.

Reply:
Done.